pkg/config/validation.go

```diff
@@ -36,3 +42,67 @@ func Validate(pArgs *ProgArgs) error {
 
 	return nil
 }
+
+func validateConfigLetPath(configletDir, configletName string) (string, error) {
+	configletPath, err := filepath.EvalSymlinks(filepath.Clean(filepath.Join(configletDir, configletName)))
+	if err != nil {
+		return "", err
+	}
+	if filepath.Dir(configletPath) != configletDir {
+		return "", fmt.Errorf("configlet %q is not within %q", configletName, configletDir)
```

Comment:
This is possible only in case of softlinks, so maybe we should imply that in the error.